web-client/src/presenter/actions/TrialSessionWorkingCopy/getTrialSessionWorkingCopyDataAction.js

```diff
@@ -0,0 +1,13 @@
+import { state } from 'cerebral';
+
+/**
+ * getTrialSessionWorkingCopyDataAction
+ *
+ * @param {object} providers.get the cerebral get function
+ * @returns {Object} filters, sessionNotes
+ */
+
+export const getTrialSessionWorkingCopyDataAction = ({ get }) => {
```

Comment:
Nit: Could this be condensed into a get call in the action where these objects are used? Typically we pass `props` around from action to action in a sequence when they are being mutated. Since the action that consumes these values grabs them from props I think they could just be pulled off of state in that action to make things a bit cleaner!

Reply:
Resolved!